src/components/grid-view/GridView.js

```diff
@@ -38,36 +39,47 @@ class GridView extends React.Component<Props> {
     });
 
     componentDidUpdate(prevProps: Props) {
-        const { columnCount, width } = this.props;
+        const { columnCount, currentCollection, width } = this.props;
 
         // The React Virtualized Table must be notified when either the cached
         // row sizes or the parent width change. If omitted, rows are sized
         // incorrectly resulting in gaps or content overlap.
-        if (columnCount !== prevProps.columnCount || width !== prevProps.width) {
+        if (
+            columnCount !== prevProps.columnCount ||
```

Comment:
Previous values should be destructured in the function signature as `prevColumnCount`, etc.